modules/missionmail.js

```diff
@@ -0,0 +1,245 @@
+// @ts-check
+const u = require("../utils/utils");
+const c = require("../config/config.json");
+const receive = require("imapflow");
+const interpret = require("mailparser-mit");
+const htmlparse = require("html-to-text");
+const Augur = require("augurbot-ts");
+const { ButtonStyle } = require("discord.js");
+
+const [approveIdPrefix, rejectIdPrefix] = ["approveMissionMail", "rejectMissionMail"];
+
+const replyRegexes = [
+  // /^on[\s\n\r]+.+wrote:[\s\n\r]*$/im,
+  // /on[\s\n\r]+.+wrote:[\s\n\r]*/im,
+  /^on[^>]*@[^<]*wrote:\n\n>/im,
+  // /^on\s.+wrote:\s*$/im, :(){:|:&};:
+  // /On\s.+wrote:\s*/im,
```

Comment:
unsure if we should keep these commented regex. Also need to test them to make sure they work

(insert picture of the long email regex that is only mostly accurate)